package/MDAnalysis/core/universe.py

```diff
@@ -298,6 +298,12 @@ def __init__(self, *args, **kwargs):
         # Universes are anchors by default
         self.is_anchor = kwargs.get('is_anchor', True)
 
+    def copy(self):
```

Comment:
Yeah `a.positions` and `b.positions` will be completely independent (shallow or not) with no knowledge of each other.

`shallow` would make `a.masses = 12.0` affect `b.masses` as this data resides in the `topology` object not the reader